components/home/menu/shuttleBusSchedule/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import {
+  View, Text, ScrollView, SectionList
+} from 'react-native';
+import { ButtonGroup } from 'react-native-elements';
+import shuttleScheduleInformation from './shuttleScheduleService';
+import styles from './styles';
+
+const buttons = ['SGW', 'LOY'];
+export default class ShuttleSchedule extends Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedButtonIndex: 0
+    };
+    this.updateButtonIndex = this.updateButtonIndex.bind(this);
+  }
+
+  /** The function will return the appropriate schedule.
+   * Button index is as follows: 0 -> SGW and 1->LOY
+   * @param {Number} selectedButtonIndex - index of the button, either 0 or 1
+   */
+  getShuttleCampusInformation(selectedButtonIndex) {
+    if (selectedButtonIndex === 1) { return shuttleScheduleInformation.Mon_Thu.LOY; }
+    return shuttleScheduleInformation.Mon_Thu.SGW;
```

Comment:
Hardcoded dates for shuttle schedule (what if it's Friday or a weekend)?

Reply:
Fixed